database/src/main/java/config/DatabaseConfiguration.java

```diff
@@ -47,7 +47,7 @@ public LocalContainerEntityManagerFactoryBean entityManagerFactory() {
         properties.put("hibernate.show_sql", environment.getRequiredProperty("hibernate.show_sql"));
         properties.put("hibernate.format_sql", environment.getRequiredProperty("hibernate.format_sql"));
         properties.put("hibernate.hbm2ddl.auto", environment.getRequiredProperty("hibernate.hbm2ddl.auto"));
-        properties.put("hibernate.hbm2ddl.import_files", environment.getRequiredProperty("hibernate.hbm2ddl.import_files"));
+        //properties.put("hibernate.hbm2ddl.import_files", environment.getRequiredProperty("hibernate.hbm2ddl.import_files"));
```

Comment:
Change such cases in TODO please. Or mark them as such. Or create a Github issue for that. Not a commented line. :-1: